common/config/rush/browser-approved-packages.json

```diff
@@ -2,13 +2,18 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/approved-packages.schema.json",
   "packages": [
+
     {
       "name": "react",
       "allowedCategories": [ "tests" ]
     },
     {
       "name": "react-dom",
       "allowedCategories": [ "tests" ]
```

Comment:
Super-super-super-nit: should these be alphabetized by `name`?
```
    {
      "name": "react-dom",
      "allowedCategories": [ "tests" ]
    },
    {
      "name": "react",
      "allowedCategories": [ "tests" ]
```

Reply:
This file is machine generated. It was out of date. Refreshing it will sort the entries automatically.